<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BloodDonation</class>
 <widget class="QWidget" name="BloodDonation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Blood Donation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #b91c1c; color: white; font-weight: bold;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="topBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #b91c1c; color: white; font-weight: bold;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="menuIcon">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>≡</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleText">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Blood C+</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bellIcon">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>🔔</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="aboutUsBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #e63946;
    color: white;
    border-radius: 12px;
    padding: 8px 8px;
    font-size: 16px;
    font-weight: bold;
    border: none;
}

QPushButton:hover {
    background-color: #d62839;
}

QPushButton:pressed {
    background-color: #ba1f33;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/information.png</normaloff>../../Downloads/information.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preperationBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #e63946;
    color: white;
    border-radius: 12px;
    padding: 8px 8px;
    font-size: 16px;
    font-weight: bold;
    border: none;
}

QPushButton:hover {
    background-color: #d62839;
}

QPushButton:pressed {
    background-color: #ba1f33;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/clipboard.png</normaloff>../../Downloads/clipboard.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="whyImportantBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #e63946;
    color: white;
    border-radius: 12px;
    padding: 8px 16px;
    font-size: 16px;
    font-weight: bold;
    border: none;
}

QPushButton:hover {
    background-color: #d62839;
}

QPushButton:pressed {
    background-color: #ba1f33;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/warning.png</normaloff>../../Downloads/warning.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="whoCanDonateBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #e63946;
    color: white;
    border-radius: 12px;
    padding: 8px 16px;
}

QPushButton:hover {
    background-color: #d62839;
}

QPushButton:pressed {
    background-color: #ba1f33;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/blood-drop.png</normaloff>../../Downloads/blood-drop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>About us</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Preparation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Why important?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Who can donate?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QWidget {
    border-radius: 10px;
    padding: 8px 2px;
    font-weight: bold;
    border: none;
	color: rgb(0,0,128);
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-215</y>
        <width>382</width>
        <height>598</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="scrollLayout">
       <item>
        <widget class="QFrame" name="bloodCard">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="cardLayout">
          <item>
           <widget class="QLabel" name="emergencyTitle">
            <property name="font">
             <font>
              <pointsize>19</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0,0,128);</string>
            </property>
            <property name="text">
             <string>EMERGENCY BLOOD REQUIRED</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bloodTitle">
            <property name="styleSheet">
             <string notr="true">color: rgb(0,0,128);</string>
            </property>
            <property name="text">
             <string>AB+ BLOOD REQUIRED</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="patientInfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(0,0,128);</string>
            </property>
            <property name="text">
             <string>Patient Name: Talasbekov Doolot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bagsInfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(0,0,128);</string>
            </property>
            <property name="text">
             <string>No. of Bags: 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="transfusionInfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 128);</string>
            </property>
            <property name="text">
             <string>08 Apr 2025 08:00 AM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hospitalInfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 128);</string>
            </property>
            <property name="text">
             <string>Hospital: Amanat Hospital </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="locationInfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 128);</string>
            </property>
            <property name="text">
             <string>Location: Bishkek, Turusbekova 881</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="impactImage">
         <property name="styleSheet">
          <string notr="true">color: rgb(9, 9, 9);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Downloads/images.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="bargraph">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomNav">
     <item>
      <widget class="QPushButton" name="donateBtn">
       <property name="styleSheet">
        <string notr="true">/* Общий стиль для всего окна */
QWidget {
    background-color: #b30000; /* насыщенный красный */
    font-family: 'Segoe UI', sans-serif;
    color: white;
}

/* Заголовки */
QLabel#titleLabel {
    font-size: 24px;
    font-weight: bold;
    color: white;
}

/* Подзаголовки и текст */
QLabel {
    font-size: 14px;
    color: white;
}

/* Поля ввода */
QLineEdit {
    background-color: white;
    border: 2px solid #ff4d4d;
    border-radius: 10px;
    padding: 6px;
    color: #333;
    font-size: 14px;
}

/* Фокус на поле */
QLineEdit:focus {
    border: 2px solid #ff1a1a;
}

/* Кнопки */
QPushButton {
    background-color: #b30000;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border-radius: 12px;
    padding: 8px 16px;
    border: 2px solid white;
}

QPushButton:hover {
    background-color: #ffcccc;
    color: #800000;
}

QPushButton:pressed {
    background-color: #ffe6e6;
    border: 2px solid #ff9999;
}

/* Ошибки или уведомления */
QLabel#errorLabel {
    color: #ffcccc;
    font-size: 12px;
}

/* Кнопки внизу: Sign In / Sign Up переключение */
QPushButton#switchButton {
    background: transparent;
    color: white;
    font-size: 12px;
    text-decoration: underline;
    border: none;
}

QPushButton#switchButton:hover {
    color: #ff9999;
}</string>
       </property>
       <property name="text">
        <string>Donate blood</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="requestBtn">
       <property name="styleSheet">
        <string notr="true">/* Общий стиль для всего окна */
QWidget {
    background-color: #b30000; /* насыщенный красный */
    font-family: 'Segoe UI', sans-serif;
    color: white;
}

/* Заголовки */
QLabel#titleLabel {
    font-size: 24px;
    font-weight: bold;
    color: white;
}

/* Подзаголовки и текст */
QLabel {
    font-size: 14px;
    color: white;
}

/* Поля ввода */
QLineEdit {
    background-color: white;
    border: 2px solid #ff4d4d;
    border-radius: 10px;
    padding: 6px;
    color: #333;
    font-size: 14px;
}

/* Фокус на поле */
QLineEdit:focus {
    border: 2px solid #ff1a1a;
}

/* Кнопки */
QPushButton {
    background-color: #b30000;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border-radius: 12px;
    padding: 8px 16px;
    border: 2px solid white;
}

QPushButton:hover {
    background-color: #ffcccc;
    color: #800000;
}

QPushButton:pressed {
    background-color: #ffe6e6;
    border: 2px solid #ff9999;
}

/* Ошибки или уведомления */
QLabel#errorLabel {
    color: #ffcccc;
    font-size: 12px;
}

/* Кнопки внизу: Sign In / Sign Up переключение */
QPushButton#switchButton {
    background: transparent;
    color: white;
    font-size: 12px;
    text-decoration: underline;
    border: none;
}

QPushButton#switchButton:hover {
    color: #ff9999;
}</string>
       </property>
       <property name="text">
        <string>Request blood</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
